src/components/ListingDetail/ListingDetail.js

```diff
@@ -1,18 +1,49 @@
 import { getListing } from '../../api/glue-api'
 import { mapGetters } from 'vuex'
+import VueSlickCarousel from 'vue-slick-carousel'
+import 'vue-slick-carousel/dist/vue-slick-carousel.css'
+// optional style for arrows & dots
+import 'vue-slick-carousel/dist/vue-slick-carousel-theme.css'
 
 export default ({
   name: 'ListingDetail',
 
+  components: {
+    VueSlickCarousel,
+  },
+
   data() {
     return {
       isLoading: true,
-      listing: {}
+      listing: {},
+      settings: {
+        arrows: true,
+        // dots:true,
+        // dotsClass: 'slick-dots custom-dot-class',
+        edgeFriction: 0.35,
+        infinite: true,
+        speed: 500,
+        slidesToShow: 1,
+        slidesToScroll: 1,
+      },
     }
   },
-  
+
   computed: {
     ...mapGetters(['getHrefFromId']),
+
+    advertiserLogoUrl() {
+      let url = this.listing.account.logoUrl
+      url = url.replace('/{action}/{width}x{height}/', '/fit-in/136x94/')
```

Comment:
This replace logic could be a specialist function, couldn't?
something like:

```js
function normalizeImageUrl(url, replaceValue) {
  return url.replace('{action}/{width}x{height}', `${replaceValue}`)
}

// using

...
    advertiserLogoUrl() {
      const url = this.listing.account.logoUrl
      return normalizeImageUrl(url, 'fit-in/136x94')
    }

...

    imageURL() {
      const medias = this.listing.medias
      medias.forEach(media => {
        media.url = normalizeImageUrl(media.url., 'fit-in/800x360')
      })

      this.listing.medias = medias
    }
```

Reply:
Like a utils.js file?
Put it inside helpers folder.